frontend/http/writer_test.go

```diff
@@ -1,15 +1,12 @@
-// Copyright 2016 The Chihaya Authors. All rights reserved.
-// Use of this source code is governed by the BSD 2-Clause license,
-// which can be found in the LICENSE file.
-
 package http
 
 import (
 	"net/http/httptest"
 	"testing"
 
-	"github.com/chihaya/chihaya/tracker"
 	"github.com/stretchr/testify/assert"
```

Comment:
We should either use `..testify/require` to actually make the tests fail or use the stdlib testing everywhere (I know you didn't use `assert` or `require` in most of the new stuff, so I'm bringing this up for consistency)